LeetCode/Array/Remove_Duplicates_from_Sorted_Array/HyeonJeong.py

```diff
@@ -0,0 +1,11 @@
+class Solution:
+    def removeDuplicates(self, nums: List[int]) -> int:
+        i = 0
+        while 1:
+            if i+1 >= len(nums): # 중복되는 값을 지우다가 없는 인덱스에 접근하지 않게 종료
+                break
+            elif nums[i] == nums[i+1]: # 중복되는 값 확인
```

Comment:
5번라인의 조건문에 부합하면 어차피 while문에서 탈출하기 때문에
여기서는 그냥 if문을 사용해도 되지 않을까요?
```suggestion
            if nums[i] == nums[i+1]: # 중복되는 값 확인
```